frontend/packages/topology/src/components/page/TopologyPage.tsx

```diff
@@ -34,7 +32,7 @@ interface TopologyPageProps {
   defaultViewType?: TopologyViewType;
 }
 
-const TopologyPage: React.FC<TopologyPageProps> = ({
+export const TopologyPage: React.FC<TopologyPageProps> = ({
```

Comment:
Same problem for this guy, but I can't put a comment on line 90 🙂

Reply:
Updated. Moved blue box to below project dropdown in both places.